bitmovin/resources/models/encodings/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .muxings import Muxing, FMP4Muxing, MP4Muxing, TSMuxing, WebMMuxing, ProgressiveTSMuxing, MuxingStream, \
     ByteRange, MuxingInformationAudioTrack, MuxingInformationVideoTrack, ProgressiveTSInformation, \
     MP4MuxingInformation, ProgressiveMOVMuxing
+from .muxings.broadcast_ts import *
```

Comment:
why you are using a wildcard import here? the other imports in this file look completely different?